cli/cmd/encore/cmdutil/cmdutil.go

```diff
@@ -116,3 +117,14 @@ func DisplayError(out *os.File, err []byte) {
 
 	_, _ = os.Stderr.Write([]byte(errList.Error()))
 }
+
+var Newline string
+
+func init() {
+	switch runtime.GOOS {
+	case "windows":
+		Newline = "\r\n"
+	default:
+		Newline = "\n"
+	}
+}
```

Comment:
nit; I'd probably written this in `cmdutil_windows.go` and `cmdutil_nix.go` and used build tags, but this is fine :)